<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_calib">
              <property name="minimumSize">
               <size>
                <width>108</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>108</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Sweep</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fnameCalib">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="browseCalib">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string> Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_header">
              <property name="minimumSize">
               <size>
                <width>108</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>108</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Header</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fnameHead">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="browseHead">
              <property name="text">
               <string> Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_I">
              <property name="minimumSize">
               <size>
                <width>108</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>108</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>I file   </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fnameI">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="browseI">
              <property name="text">
               <string> Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_Q">
              <property name="minimumSize">
               <size>
                <width>108</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>108</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Q file</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fnameQ">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="browseQ">
              <property name="text">
               <string> Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_generate_random_signal">
              <property name="text">
               <string>PLOT CALIB</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="headerButton">
              <property name="text">
               <string>LOAD HEADER</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="fitButton">
              <property name="text">
               <string>FIT</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearButton">
              <property name="text">
               <string>CLEAR</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="exit_button">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>60</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>EXIT</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Param√®tres</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="7" column="0">
        <widget class="QLabel" name="labelFres_11">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>tau</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLabel" name="labelPhi0">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="labelTau">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="labelFres">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelFres_8">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>Phi0</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="labelQl">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="labelFres_4">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>k</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="labelQc">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="labelQi">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="labelFres_9">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>k</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>Hz</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>Hz</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="labelTone">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelFres_5">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>Qc</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="labelFres_14">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>a</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QLabel" name="labelFres_10">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>rd</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QLabel" name="labelFres_13">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>s</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>Ql</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelFres_7">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>Qi</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="labelFres_6">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>k</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QLabel" name="labelFres_16">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>no unit</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>TONE</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labelRun">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>Fres</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>RUN</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="labelFres_17">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>alpha</string>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QLabel" name="labelFres_19">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>rd</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QLabel" name="labelAlpha">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLabel" name="labelA">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" rowspan="2">
        <widget class="MplWidget" name="MplWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>420</width>
           <height>420</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="MplWidget" name="MplWidget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="MplWidget" name="MplWidget_3" native="true">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>967</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>exit_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>701</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>FileSelected(QString)</signal>
  <slot>browse()</slot>
  <slot>PrintFile()</slot>
  <slot>plot_data()</slot>
 </slots>
</ui>
